src/components/extensive/WizardForm/utils.ts

```diff
@@ -46,10 +46,19 @@ export const getSchemaFields = (fields: FormField[]) => {
         }
       });
     } else {
-      schema[field.name] = field.validation?.nullable().label(" ");
+      if (!field.validation) {
```

Comment:
I won't require you change this after the fact, but please remember to use explicit null checks: `if (field.validation == null)`